### PR TITLE
BAH-2273|BAH-1995 text-added-in-the-history-examination-image-is-not-…

### PR DESCRIPTION
**Pull request by syadav123**
…getting-displayed-properly-in-the-patient-dashboard

### CODE REVIEW

**Comment by gsluthra** [comment]
Is desk-check done for this PR with @gokultw ?

**Comment by gokultw** [comment]
> Is desk-check done for this PR with @gokultw ?

Desk-check happened last month @gsluthra, the screenshots in the recent comment in JIRA look something different. Will connect with him.

**Comment by gsluthra** [comment]
Any update on this ticket @gokultw ?

**Comment by gokultw** [comment]
Desk Check comments-

- Existing notes view is impacted in all display controls due to this PR -

image

- Notes for consultation images can be removed from History & examination display control and Notes icon should be displayed instead of text if the image is having the Notes.

image

File: ui/app/common/obs/views/showObservation.html [hunk 1]
```diff
@@ -25,10 +25,10 @@
...
               ng-if="::(observation.isFormElement() && !observation.isImageConcept() && !observation.isVideoConcept())">
-            <pre class="left obs-label" title="{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}"
-                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')>-1">{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}</pre>
-            <pre class="left obs-label" title="{{::observation.getDisplayValue() | titleTranslate}}"
-                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')===-1">{{::observation.getDisplayValue() | titleTranslate}}</pre>
+            <span class="left obs-label-file" title="{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}"
+                ng-if="configIsObservationForImages && observation.getDisplayValue().lastIndexOf('__')>-1">{{::observation.getDisplayValue().substring(observation.getDisplayValue().lastIndexOf('__')+2) | titleTranslate}}</span>
```
Comment (gsluthra) [inline ui/app/common/obs/views/showObservation.html:29]: This is a really long if() statement. Can we extract this into observation class, and add a more readable method name.. like observation.isValidImage( ) or something like that, which is more meaningful as to WHY this if statement exists.

**Review by gsluthra — APPROVED** [review]

**Pull request merged**